changelogs/3.0.0.md

```diff
@@ -0,0 +1,478 @@
+  `box.schema.user.grant`. Now only users with the `'execute'` privilege
+  granted on `sql` or `universe` can execute SQL expressions with the
+  `IPROTO_EXECUTE` or `IPROTO_PREPARE` requests. To revert to the old behavior
+  (no SQL access checks), use the `sql_priv` compat option (gh-8803).
+* Added the `leader_name` field to `box.info.election` (gh-8931).
+* Now it is possible to specify the sort order for each part of an index
+  (gh-5529).
+* Added support for sending tuple formats in IPROTO responses. Added a
+  `box_tuple_extension` backward compatibility option to disable sending
+  tuple formats in responses to IPROTO call and eval requests (gh-8146).
+* Added support for sending tuple formats in IPROTO call and eval request
+  arguments. `box_tuple_extension` backward compatibility option can be used to
+  disable receiving tuple formats in IPROTO call and eval request arguments
+  (gh-8633).
+* Added a new `is_sync` parameter to `box.atomic()`. To make the transaction
+synchronous, set the `is_sync` option to `true`. Setting `is_sync = false` is
+prohibited. If any value other than true/nil is set, for example
+`is_sync = "some string"`, then an error will be thrown (gh-8650).
+
+### Cli
+
+* **[Behavior change]** Disabled starting the Lua REPL by default when running
+  Tarantool. Now, Tarantool yields the message that shows the command usage.
+  To run the Lua REPL, just set the `-i` flag. To pass a Lua script contents via
+  `stdin`, use dash (`-`) as the script name. For more information see a help
+  message by running `tarantool -h` (gh-8613).
+* Introduced new command-line options `--name` and `--config` (gh-8613).
+
+### Compat
+
+* The following compatibility options' defaults were switched to new behavior:
+    * `yaml_pretty_multiline`
+    * `sql_seq_scan_default`
+    * `json_escape_forward_slash`
+    * `fiber_channel_close_mode`
+    * `fiber_slice_default`
+    * `box_cfg_replication_sync_timeout`
+    * `c_func_iproto_multireturn`
+
+  More information on the new behavior can be found on the [Module compat](https://www.tarantool.io/en/doc/latest/reference/reference_lua/compat/) page.
+
+### Console
+
+* Now multiline commands can be used by setting continuation marker
+  using the `\set continuation` command.
+
+### Fiber
+
+* Exported `fiber_set_name_n`, `fiber_name`, `fiber_id`, `fiber_csw` and
+  `fiber_find` into the public C API and usable via FFI as well.
+* Make `fiber_set_joinable`, `fiber_set_ctx` and `fiber_get_ctx`
+  treat the NULL argument as the current fiber.
+
+### Space
+
+* Introduces space type: a new space definition field.
+* Introduces the fully temporary space type. It is the same as data-temporary
+  but also has temporary metadata. Temporary spaces can now be created in
+  read_only mode, they disappear after server restart and don't exist on
+  replicas (gh-8323).
+
+### Test
+
+#### Fuzz
+
+* Added a grammar-based SQL fuzzer (gh-4826).
```

Comment:
How about moving it down, after 'build'?

Reply:
done